var/spack/repos/builtin/packages/z3/package.py

```diff
@@ -15,7 +15,9 @@ class Z3(CMakePackage):
     url = "https://github.com/Z3Prover/z3/archive/z3-4.5.0.tar.gz"
 
     license("MIT")
+    maintainers("vmiheer")
```

Comment:
```suggestion
    maintainers("vmiheer")
    
    license("MIT")
```

 In the common format `license()` is typically placed below `maintainers()` with a blank space between the two. 

For example, 
https://github.com/spack/spack/blob/73dae78ab3aa23257fd591100cdcff0afa79b5a1/var/spack/repos/builtin/packages/gcc/package.py#L34-L36